File: c++/test18.cpp
/*
 * File: sum_average.cpp
 * Task: Calculate sum and average of vector elements
 * 
 * ## **Exercise 9: Sum of Vector (Simple - Vectors & Accumulation)**

**Type:** Common Test Scenario

**Task:**
Write a program that:
1. Creates a vector: `{10, 25, 3, 42, 18, 7, 33}`
2. Calculates the sum of all elements
3. Calculates the average (as a double)
4. Prints both

**Expected Output:**
```
Sum: 138
Average: 19.7143
 */

#include <iostream>
#include <vector>
using namespace std;

int main() {
    vector<int> numbers = {10, 25, 3, 42, 18, 7, 33};
    
    int sum = 0;
    double avg = 0;
    
    for (int i = 0; i < numbers.size(); i++) {
        sum = sum + numbers[i];
    }
    avg = (double)sum / numbers.size();
    
    cout << sum << endl;
    cout << avg << endl;

    return 0;
}

File: python/test18.py
"""
Write a function that takes a list of numbers and returns their sum.

Example:
sum_list([1, 2, 3, 4, 5]) → 15
"""

def sum_list(numbers):
    # Your code here
    sum = 0
    for number in numbers:
        sum += number
    return sum

# Test
print(sum_list([1, 2, 3, 4, 5]))  # Should print 15